ci/kind/test-e2e-kind.sh

```diff
@@ -191,20 +217,31 @@ function run_test {
   else
       go test -v -timeout=75m antrea.io/antrea/test/e2e -provider=kind --logs-export-dir=$ANTREA_LOG_DIR --skip=$skiplist
   fi
-  $TESTBED_CMD destroy kind
+  if [[ $setup_only == "false" ]];then
```

Comment:
Do we still need to do cleanup here? could it just check the parameters in quit function?

Reply:
done, removed this after refine `quit`.